src/stories/logo-link/logo-link.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { LogoLink } from './logo-link';
```

Comment:
Looks like that you can import it from `../../shared` folder and do not create `logo-link.tsx` file. It also applies to other filed in `stories` folder